### ISSUE TITLE
Pressing Ctrl+Tab in popup modal should only cycle through in the modal

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#5312]
**Did you search for similar issues before submitting this one?**
Yes but didn't find

**Describe the issue you encountered:**
Pressing Ctrl+Tab cycles through URL bar and typing in the URL bar doesn't dismiss the popup modal

**Expected behavior:**
Ctrl+Tab/Ctrl+Shift+Tab should only cycle through the fields/buttons in the modal. Focus should not be lost from the modal to URL bar

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version:
0.12.8 Preview 2 (Verified on 0.12.5 onwards)

- Steps to reproduce:
    1. Open about:autofill
    2. Click on Add Credit Card
    3. Press Ctrl+Shift+Tab when the focus is on the Name filed
    4. Focus is set to URL bar and user can enter URL
    5. On typing in URL and pressing return key, modal doesn't get removed but still remain in focus

 Selection only visible in URL mode and not in Title mode although focus is set to the URL Bar but typing doesn't have any effect

- Screenshot if needed:
![modaltab](https://cloud.githubusercontent.com/assets/17010094/19890996/e61008ca-a035-11e6-823b-209511e135c8.gif)

- Any related issues:

**Comment by bsclifton** [comment]
This is definitely a nice to have (for multiple reasons)... but I don't think it's required anytime soon.  Removing milestone
